src/tests/reducers/articleReducer.test.js

```diff
@@ -52,10 +57,32 @@ describe('articlesReducer', () => {
       userArticlesPayload: {},
       likeDislikeSuccess: false,
       likeDislikeError: {},
+      editArticleSuccess: false,
     });
   });
 
-  it('should add CREATE_ARTICLE_SUCCESS to true when an article is posted successfully', () => {
+  it('should set the loader when GET_ALL_ARTICLES_INITIATED is true', () => {
+    const action = {
+      type: GET_ALL_ARTICLES_INITIATED,
+      payload: true,
+    };
+    const currentState = articlesReducer(initialState, action);
+    expect(currentState).toEqual({
```

Comment:
Everyone seems to be doing this which is making this file ambiguous. How about this:

expect(...).toEqual({
    ...initialState,
    loading: action.payload,
});
@samuelbwambale . If it works, advise the next victims.

Reply:
I have refactored this and shared with the team